### DISCUSSION TITLE
docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181

### DISCUSSION THREAD

**Discussion by xde012** [Security-Onion-Solutions/securityonion#9063] [General]
Hi,

I am running "sudo bash so-setup-network" following the steps from https://docs.securityonion.net/en/2.3/installation.html?highlight=centos#installation-on-ubuntu-or-centos
I am installing on a CentOS 7 and it has a proxy which works because I have access to the internet from CentOS itself.

When running so-setup-network and configuring everything correctly, the installation starts, but an error occurs:
"_Summary for local
-------------
Succeeded: 48 (changed=4)
Failed:     1
-------------
Total states run:     49
Total run time:  123.897 s
gpg: clave FE507013: "Security Onion Solutions, LLC <info@securityonionsolutions.com>" sin cambios
gpg: Cantidad total procesada: 1
gpg:              sin cambios: 1
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181
Error response from daemon: Get "https://ghcr.io/v2/": proxyconnect tcp: EOF
Results:  (1)
Command failed with exit code 1; will retry in 60 seconds (1 / 5)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181
Error response from daemon: Get "https://ghcr.io/v2/": proxyconnect tcp: EOF
Results:  (1)
Command failed with exit code 1; will retry in 60 seconds (2 / 5)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181
Error response from daemon: Get "https://ghcr.io/v2/": proxyconnect tcp: EOF
Results:  (1)
Command failed with exit code 1; will retry in 60 seconds (3 / 5)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181
Error response from daemon: Get "https://ghcr.io/v2/": proxyconnect tcp: EOF
Results:  (1)
Command failed with exit code 1; will retry in 60 seconds (4 / 5)...
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.181
Error response from daemon: Get "https://ghcr.io/v2/": proxyconnect tcp: EOF
Results:  (1)
Command failed with exit code 1; will retry in 60 seconds (5 / 5)...
Command continues to fail; giving up.
Could not pull so-acng:2.3.181, please ensure connectivity to ghcr.io
Errors detected during setup; skipping post-setup steps to allow for analysis of failures._"

The URL is accessible even from any browser behind proxy. Getting this message:
_{"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}_
Something logical knowing that it is a docker repository.

How can I solve this error?
In theory the script is automated to log in against ghcr.io, is that right?

Thank you!

**Comment by dougburks** [comment]
> In theory the script is automated to log in against ghcr.io, is that right?

No, ghcr.io is a public container registry and it shouldn't require authentication. 

I suspect the authentication prompt is coming from your proxy.
